services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # TODO: Please set your custom image here and adapt the Dockerfile/entrypoint.sh accordingly
-FROM nginx:1.21.0-alpine as production 
+FROM joseluisq/static-web-server:2.0.2-alpine as production 
```

Comment:
minor: would that not mean a new Major version? or at least minor?

Reply:
I'd say minor. As the functionality per see is still the same. It only has more additions to it and bug fixes.

Reply:
After thinking about it more, I'd go for a major change here! going for `1.0.0`